src/assets/i18n/en.json

```diff
@@ -84,6 +85,7 @@
   "login.create.assignment.description": "You need to create a profile before you can create a job",
   "login.apply.assignment.title": "Search jobs",
   "login.apply.assignment.description": "You need to create a profile before you can search for a job",
+  "start.or": "or",
```

Comment:
If it was all going to be a string I understand, but according to the design it's:
<btn (with text)> or <btn (with text)> and I didn't think it was possible to put a translation around the whole thing, is there some smart way to do it? :) Or should I simply remove the "or" from the design? (This design is a copy of the create/sign in from the job-details page)